dataprofiler/tests/profilers/test_unstructured_labeler_profile.py

```diff
@@ -131,7 +131,25 @@ def test_profile(self, processor_class_mock, model_class_mock):
 
         # key and value populated correctly
         self.assertDictEqual(expected_profile, profile)
+
+
+    def test_unstructured_labeler_profile_add(self):
```

Comment:
The test should probably be a bit more complect with more labels. Technically you could make a profile and just set the values to test the merge.

Reply:
Also we should avoid actually running the tensorflow model as the results change if the model changes

Reply:
^ Adding more labels to the test